Vic2ToHoI4/Data_Files/Fronter/fronter-options.txt

```diff
@@ -181,4 +181,14 @@ option = {
 			default = false
 		}
 	}
+}
+option = {
+	name = output_name
+	displayName = MODNAME
+	tooltip = MODNAMETIP
+	textSelector = {	
+		value = "myMod"
```

Comment:
Keep it empty. We don't want to change mod names if not needed/requested by user.